src/main/java/org/broadinstitute/hellbender/utils/Utils.java

```diff
@@ -1137,4 +1137,83 @@ public static void forceJVMLocaleToUSEnglish() {
         final List<T> sorted = values.stream().sorted().collect(Collectors.toList());
         return sorted.get(sorted.size() / 2);
     }
+
+    /**
+     * Splits a String using indexOf instead of regex to speed things up.
+     * If given an empty delimiter, will return each character in the string as a token.
+     * @param str the string to split.
+     * @param delimiter the delimiter used to split the string.
+     * @return A {@link List} of {@link String} tokens.
+     */
+    public static List<String> split(final String str, final String delimiter) {
+        // This is 10 because the ArrayList default capacity is 10 (but private).
+        return split(str, delimiter, 10);
+    }
+
+    /**
+     * Splits a String using indexOf instead of regex to speed things up.
+     *
+     * @param str the string to split.
+     * @param delimiter the delimiter used to split the string.
+     * @return A {@link List} of {@link String} tokens.
+     */
+    public static List<String> split(final String str, final char delimiter) {
+        final List<String> tokens = ParsingUtils.split( str, delimiter );
+
+        // We must adjust for splitting on the last character so that the results here will be the same as
+        // the results of String.split:
+        if ( delimiter == str.charAt(str.length() - 1)) {
```

Comment:
I'm not sure if there are strict rules on this, but the formatting of the code in this comment differs from the most of the code I've seen in GATK:

- most GATK code doesn't have spaces inside parens (I see that spaces inside parens is not done consistently here as well, e.g. on this line there is a space before `delimiter` but not after `1)`)
- most GATK code formats `else` as `} else {` or `} else if (...) {`

Reply:
I prefer the extra whitespace (the space inconsistency in the parens was a typo).

I've had to maintain a lot of poorly formatted code and having some extra line breaks makes it easier to visually differentiate between different blocks.